client/my-sites/site-settings/section-writing.jsx

```diff
@@ -20,9 +20,9 @@ module.exports = React.createClass( {
 	render: function() {
 
 		return (
-			<Card className="writing-settings">
-				<WritingForm site={ this.props.site } />
-			</Card>
+
+			<WritingForm site={ this.props.site } />
```

Comment:
In cases where we have a single return value that isn't a long line, I personally prefer to eliminate the parentheses.

```js
render: function() {
	return <WritingForm site={ this.props.site } />;
}
```